.eslintignore

```diff
@@ -1,2 +1,3 @@
 **/*.html
 build/*
+coverage/*
```

Comment:
Is this something that is generated by the package or your local environment?

Reply:
if you run tests with coverage, yes